format/avro/decoders/long.go

```diff
@@ -0,0 +1,33 @@
+package decoders
+
+import (
+	"github.com/wader/fq/format/avro/schema"
+	"github.com/wader/fq/pkg/decode"
+	"github.com/wader/fq/pkg/scalar"
+)
+
+const intMask = byte(127)
+const intFlag = byte(128)
+
+// VarZigZag reads a variable length zigzag long from the current position in decoder
+func VarZigZag(d *decode.D) int64 {
```

Comment:
Error if more than 8 bytes? btw i added support for bigint in master yesterday if that is interesting for avro

Reply:
Good idea, set it to fatal on over 8 bytes